mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/addons/fullscreen/config.json

```diff
@@ -0,0 +1,17 @@
+[{
```

Comment:
The fullscreen addon is not yet an official one.
Where does this one come from ?